libraries/AP_BoardConfig/IMU_heater.cpp

```diff
@@ -111,4 +111,20 @@ void AP_BoardConfig::set_imu_temp(float current)
 #endif
 }
 
+bool AP_BoardConfig::get_board_hearter_temp(float &temp) const {
```

Comment:
```suggestion
bool AP_BoardConfig::get_board_heater_temp(float &temp) const
{
```

similarly elsewhere for the `{` - its the style in this file